src/controllers/articleRatingControllers.js

```diff
@@ -63,5 +63,36 @@ class ArticleRatingManager {
       });
     }
   }
+
+  /**
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {message} That article is not available
+ */
+  static async ratingAverage(req, res) {
+    const averageRating = await Rating.findAll({
+      where: { articleId: req.params.articleId },
+      attributes: ['rating'],
+      raw: true,
+    });
+    if (averageRating.length === 0) {
+      return res.status(400).send({
+        message: 'This Article is Not found or it has not been rated yet,Please try agai'
```

Comment:
The message should only reflect the article not having been rated. 'Try again' is not necessary